src/hark_lang/hark_parser/nodes.py

```diff
@@ -51,18 +51,14 @@ class N_Call(Node):
 
 
 @dataclass
-class N_Async(Node):
-    expr: str
-
-
-@dataclass
-class N_Await(Node):
-    expr: list
+class N_Binop(Node):
+    lhs: Any
```

Comment:
I believe that these have to be instances of Node, not Any. Again, my bad code originally.